src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -2011,6 +2011,18 @@ int sockinfo_tcp::bind(const sockaddr *__addr, socklen_t __addrlen)
 		return -1; //error
 	}
 
+	if (!m_bound.is_anyaddr() && !m_bound.is_local_loopback()) {
```

Comment:
@igor-ivanov I think here you should also verify `m_so_bindtodevice_ip != 0` so we are sure the routing will not change later with TCP connect() call